nixos/modules/config/fonts/fontdir.nix

```diff
@@ -10,6 +10,9 @@ let
     find ${toString config.fonts.fonts} -regex "$font_regexp" \
       -exec ln -sf -t "$out/share/X11/fonts" '{}' \;
     cd "$out/share/X11/fonts"
+    ${if config.fonts.fontDir.decompressFonts then ''
```

Comment:
You could use [lib.optionalString](https://nixos.org/manual/nixpkgs/stable/#function-library.strings.optionalString) here.

Reply:
Fixed, thanks for letting me know about that function